tools/SchemaManager.Core/ISchemaManager.cs

```diff
@@ -18,28 +18,25 @@ public interface ISchemaManager
     /// Applies SQL schemas specified by the range in <paramref name="type"/>.
     /// </summary>
     /// <param name="connectionString">connection string to the backing SQL server.</param>
-    /// <param name="server">URL to the service.</param>
     /// <param name="type">The schema version to apply.</param>
     /// <param name="token">A cancellation token.</param>
     /// <returns>A task.</returns>
-    public Task ApplySchema(string connectionString, Uri server, MutuallyExclusiveType type, CancellationToken token = default);
```

Comment:
Do you know why `connectionString` is part of each method's signature and not part of the state of the schema manager? I'm not aware of schema manager communicating with different SQL servers

Reply:
Addressed in latest commit(s).